keyboards/haverworks/theseus75/config.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2023 Moritz Plattner (@ebastler), Alex Havermale (@haversnail)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Defines for configuring the serial driver for split comms (see https://docs.qmk.fm/drivers/serial) */
+#define SERIAL_USART_DRIVER SD3    // USART 3
+#define USART3_REMAP               // Remap USART TX and RX pins for half duplex
+#define SERIAL_USART_TX_PAL_MODE 7 // Pin alternate function 7 = USART3_TX for PB10
+
+/* Defines for required pins */
+#define ID_PIN A13
+#define USBSW_PIN A14     // Switches the hub input/output with the MUXes
+#define PSW_PIN A15       // Turns the power routing to the USB-C port on/off
+#define SPLIT_HAND_PIN B9 // The MCUs probe a pin to determine if they are left or right side. high = left, low = right
+#define BUS_B_PIN C13     // Tells the hub to be bus-powered or self-powered (which in turn tells the clients to use 100 mA or 500 mA)
+#define USB_VBUS_PIN C15  // Only the master side will have VBUS present at power-up (used to determine master/slave)
+#define USBPD_1_PIN F0
+#define USBPD_2_PIN F1
+
+/* Additional defines for managing power and state scross split */
+#define DISABLE_BUS_POWER_MODE TRUE
+#define KB_STATE_SYNC_INTERVAL 500
+#define USBPD_ALLOWANCE_CHECK_INTERVAL 100
+#define SPLIT_TRANSACTION_IDS_KB RPC_ID_KB_STATE
+
+/* Defines for the RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+#    define INDICATOR_MAX_BRIGHTNESS 255
+#    define CAPS_LOCK_LED_INDEX 0 // LED1 on PCB
+#    define WS2812_EXTERNAL_PULLUP
+#    define WS2812_PWM_DRIVER PWMD3
+#    define WS2812_PWM_CHANNEL 4
+#    define WS2812_PWM_PAL_MODE 10
+#    define WS2812_DMA_STREAM STM32_DMA1_STREAM2
+#    define WS2812_DMA_CHANNEL 2
+#    define WS2812_DMAMUX_ID STM32_DMAMUX1_TIM3_UP
+#endif
```

Comment:
Doesn't really need to be guarded
```suggestion
#define INDICATOR_MAX_BRIGHTNESS 255
#define CAPS_LOCK_LED_INDEX 0 // LED1 on PCB
#define WS2812_EXTERNAL_PULLUP
#define WS2812_PWM_DRIVER PWMD3
#define WS2812_PWM_CHANNEL 4
#define WS2812_PWM_PAL_MODE 10
#define WS2812_DMA_STREAM STM32_DMA1_STREAM2
#define WS2812_DMA_CHANNEL 2
#define WS2812_DMAMUX_ID STM32_DMAMUX1_TIM3_UP
```

Reply:
You're right – resolved in 45929fc.